common-rendering/src/components/FirstPublished.tsx

```diff
@@ -66,6 +73,23 @@ const FirstPublished = ({
 					publishedDate.getMinutes(),
 				)}`}
 			</span>
+			{isOriginalPinnedPost && (
+				<span
+					css={css`
+						width: 0.875rem;
```

Comment:
I wonder if these should be px? we seem to be using px elsewhere in this file.

Reply:
It's a good question, I tend to favour rem where we can for scaling but use Source's space (with px) for spacing (margin, padding etc). @oliverlloyd is there any hard and fast rules for this in DCR?

Reply:
Here’s a one-button fix, @abeddow91:

```suggestion
						width: 14px;
```

Reply:
Great, thanks @oliverlloyd and @mxdvl !